services/archetypes/worker/src/index.ts

```diff
@@ -2,15 +2,16 @@ import path from 'path'
 
 import { NativeConnection, Worker as TemporalWorker, bundleWorkflowCode } from '@temporalio/worker'
 
-import { Config, Service } from '@crowd/standard'
+import { Config, Service } from '@crowd/archetype-standard'
 import { getDbConnection, DbStore } from '@crowd/database'
 
 // List all required environment variables, grouped per "component".
 // They are in addition to the ones required by the "standard" archetype.
 const envvars = {
   worker: ['CROWD_TEMPORAL_SERVER_URL', 'CROWD_TEMPORAL_NAMESPACE', 'CROWD_TEMPORAL_TASKQUEUE'],
   postgres: [
-    'CROWD_POSTGRES_HOST',
+    'CROWD_POSTGRES_READER_HOST',
```

Comment:
these are not correct env variables - check .env.dist.local for the correct ones - they start with `CROWD_DB_`

Reply:
Same for redis username and `CROWD_SERVICE` should be just `SERVICE`